src/components/Round/RoundResult.js

```diff
@@ -22,17 +23,18 @@ const RoundResult = (props) => {
       <Card className={cssClasses}>
         <ResultIdItem title={'Position'} info={props.position} />
         <ResultInfoItem title={'Points'} info={props.points} />
-        <ResultInfoItem title={'Code'} info={props.code} />
-        <ResultInfoItem title={'Name'} info={props.givenName} />
-        <ResultInfoItem title={'Last Name'} info={props.familyName} />
+        <ResultNameItem
+          title={'Name'}
+          name={props.givenName}
```

Comment:
Consider simplifying the component by passing in the full name instead of `name` and `surname`. Ideally component shouldn't know anything about how it's composed, just show 😎

Reply:
Done